tests/unit/test_package_managers.py

```diff
@@ -1086,3 +1087,133 @@ def test_check_presence(
 
     if expected_stderr:
         assert_log(caplog, remove_colors=True, message=MATCH(expected_stderr))
+
+
+def _parametrize_test_install_filesystempath() -> Iterator[
+        tuple[Container, PackageManagerClass, FileSystemPath, Optional[str], Optional[str]]]:
+
+    for container, package_manager_class in CONTAINER_BASE_MATRIX:
+        if package_manager_class is tmt.package_managers.dnf.Dnf5:
+            yield container, \
+                package_manager_class, \
+                FileSystemPath('/usr/bin/dos2unix'), \
+                r"rpm -q --whatprovides /usr/bin/dos2unix \|\| dnf5 install -y  /usr/bin/dos2unix", \
+                '[1/1] dos2unix', \
+                None  # noqa: E501
+
+        elif package_manager_class is tmt.package_managers.dnf.Dnf:
+            if 'centos:stream8' in container.url or 'fedora:rawhide' in container.url:
```

Comment:
Why is this treated with own if/else? The yield is identical, isn't it?

Reply:
A development relict. I discovered one more, fixed in fbb706ad